endpoints/openrtb2/video_auction.go

```diff
@@ -242,7 +273,13 @@ func cleanupVideoBidRequest(videoReq *openrtb_ext.BidRequestVideo, podErrors []P
 	return videoReq
 }
 
-func handleError(labels *pbsmetrics.Labels, w http.ResponseWriter, errL []error, vo *analytics.VideoObject) {
+func handleError(labels *pbsmetrics.Labels, w http.ResponseWriter, errL []error, vo *analytics.VideoObject, enableDebug bool, errorCacheID *string) {
+	if enableDebug {
+		if rawUUID, _ := uuid.NewV4(); len(rawUUID) > 0 {
+			*errorCacheID = rawUUID.String()
+		}
+		errL = append(errL, fmt.Errorf("Debug cache ID: [%s]", *errorCacheID))
```

Comment:
I'm a bit confused here. If we're sending back the cache id in the body when there's no video response, why can't we just send back the full debug info as well? If the caller can inspect the response for the id, can't they inspect the response for everything?

Reply:
Confirmed that we will be using this output for now, where just the cache ID is logged. We will keep an eye out for any requested changes regarding what should be returned in an error case.